wgsl/index.bs

```diff
@@ -6965,10 +6966,19 @@ That's not a full user-defined function declaration.
     <td>|T| is [FLOATING]
     <td class="nowrap">`reflect(`|e1|`:` |T| `, `|e2|`:` |T| `) -> ` |T|
     <td>For the incident vector |e1| and surface orientation |e2|, returns the reflection direction
-    |e1|`-2*dot(`|e2|`,`|e1|`)*|e2|`.
+    |e1|`-2*dot(`|e2|`,`|e1|`)*`|e2|.
     [=Component-wise=] when |T| is a vector.
     (GLSLstd450Reflect)
 
+  <tr algorithm="refract">
+    <td>|T| is [floating]<br>I is f32
```

Comment:
should `T` be qualified more precisely as a vector here?

Reply:
I think it may be better. Should I change the `T` in `reflect` too, since it's the same situation?

Reply:
Looks like `reflect` needs to be fixed in the spec to only work on vectors.

Reply:
Should I fix it in a new PR or just this PR?

Reply:
Would be fine by me to fix it here, in a separate commit. Not sure if @dneto0 feels differently.
Thank you!

Reply:
I would open a new PR to fix reflect for being more clear, thanks!